src/components/DashboardModule/dashboardmodule.scss

```diff
@@ -65,4 +64,23 @@
         font-size: 14px;
         display: block;
     }
-}
\ No newline at end of file
+}
+
+@media screen and (max-width:600px){
```

Comment:
Looks like a good spot for a variable of some sort - future mobile-responsive styling can build from the foundation you put in place here. 💪

Reply:
So bulma has naming conventions for responsive flexbox column layouts that we aren't really taking advantage of in our DOM structure, and I think that might be cleaner still than defining our own mobile breakpoint. I think I'll try to incorporate bulma's columns in the DOM and take advantage of columns tiling vertically on mobile by default.

Reply:
I've located an existing variable used by bulma at https://github.com/codeforpdx/dwellingly-app/blob/f17e909d8cd9eb2a7ace32daafeb02ed4d0f181b/src/scss/variables.scss#L20 that's used to define the mobile breakpoint. Currently, it's set to an unreasonably large number. In my mind, the mobile breakpoint should be set to something like 540px or maybe 720px, not 1200px as it is currently. I'll see if setting this number lower breaks or improves anything.

Reply:
Unfortunately, there's a lot of custom SCSS that's overriding the thoughtful presets that bulma provides; for the most part we may as well not even be using bulma at all, given how inconsistently we're using its conventions. As it stands, the fastest way for me to patch all the weird custom css is with quick and dirty sass breakpoints, at least for now.